ISI/experiment.js

```diff
@@ -0,0 +1,691 @@
+  timing_post_trial: 1000
+};
+
+var subjectPracticeBlock ={
+  type: 'single-stim',
+  stimuli: instructionsSetup,
+  is_html: true,
+  data: {exp_id: "ISI", trial_id: "test"},
+  choices: [37],
+  timing_post_trial: 500,
+};
+
+
+var start_test_block = {
+  type: 'text',
+  text: '<div class = centerbox><p class = block-text>A trial will look like that. There will be two conditions that affect how your reward will be counted.  In one condition </p><p class = block-text> </p></div>',
+  cont_key: 13,
+  timing_post_trial: 1000
+};
+
+
+
+var practice_block = {
+  type: 'single-stim',
+  stimuli: getRound,
+  is_html: true,
+  data: {exp_id: "ISI", trial_id: "test"},
+  choices: [37],
+  timing_post_trial: 0,
+  on_finish: appendTestData,
+};
+
+
+var rewardFW_block = {
+  type: 'single-stim',
+  stimuli:getRewardFW,
+  is_html: true,
+  data: {exp_id: "ISI", trial_id: "reward"},
+  choices: [13],
+  timing_post_trial: 1000,
+  on_finish: appendRewardDataFW,
+};
+
+var rewardDW_block = {
+  type: 'single-stim',
+  stimuli:getRewardDW,
+  is_html: true,
+  data: {exp_id: "ISI", trial_id: "reward"},
+  choices: [13],
+  timing_post_trial: 1000,
+  on_finish: appendRewardDataDW,  
+};
+
+
+
+var subjectRewardBlock ={
+  type: 'single-stim',
+  stimuli:getReward,
+  is_html: true,
+  data: {exp_id: "ISI", trial_id: "reward"},
+  choices: [13],
+  timing_post_trial: 1000,
+};
+
+
+
+var practice_chunk = {
+    chunk_type: 'while',
+    timeline: [practice_block],
+    continue_function: function(data){ 
+       if (roundOver == 2) {
+			return false
+		} else if(roundOver ==1 || roundOver==0){
+			return true
+		}
+    }
+}
+
+var reset_block = {
+    type: 'call-function',
+    func: resetRound,
+    timing_post_trial: 0
+}
+
+/* create experiment definition array */
+var ISI_experiment = [];
+ISI_experiment.push(welcome_block);
+ISI_experiment.push(instructions_block);
+ISI_experiment.push(start_practice_block);
+ISI_experiment.push(subjectPracticeBlock);
+ISI_experiment.push(subjectRewardBlock);
+ISI_experiment.push(start_test_block);
+
+
+if(whichCond==0){// do the FW first, then DW
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
+ISI_experiment.push(reset_block);
+
+ISI_experiment.push(practice_chunk);
+ISI_experiment.push(rewardFW_block);
```

Comment:
when you change the task names to lowercase, be sure to change these variables as well to lowercase, eg `isi_experiment`